### PR TITLE
Trying to port linux web framework tests to engine_v2

### PR DESCRIPTION
**Pull request by ricardoamador**
*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I signed the [CLA].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: ci/builders/linux_web_framework.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "builds": [
+        {
+            "archives": [
+                {
```
Comment (godofredoc) [inline ci/builders/linux_web_framework.json:5]: I believe you can leave archives empty, or you may even be able to remove it. The whole build will be uploaded to cas and you can reference it in the test dependencies by name.

Comment (ricardoamador) [reply ci/builders/linux_web_framework.json:5]: So if I remove archives then I can reference the created output with the build name?

Comment (ricardoamador) [reply ci/builders/linux_web_framework.json:5]: Ah yeah you can reference it in the dependencies.

**Comment by Hixie** [comment]
Note from PR triage: As of August 15, @ricardoamador said this was still on his radar.

**Comment by ricardoamador** [comment]
Closing for now.

**Pull request closed without merging**